src/test/java/org/springframework/data/couchbase/domain/ReactiveAirportRepository.java

```diff
@@ -99,6 +101,15 @@
 	@ScanConsistency(query = QueryScanConsistency.REQUEST_PLUS)
 	Mono<Airport> findByIata(String iata);
 
+	@Query("#{#n1ql.delete} WHERE #{#n1ql.filter} and  iata = $1 #{#n1ql.returning}")
```

Comment:
nit: extra whitespace between and and iata, also below

Reply:
Oof.  Formatter does not reformat quoted strings!